src/sentry/api/bases/group.py

```diff
@@ -14,18 +14,18 @@ class GroupPermission(ProjectPermission):
         'DELETE': ['event:delete'],
     }
 
-    def has_object_permission(self, request, view, group):
+    def has_object_permission(self, request, view, issue):
```

Comment:
was going to rename things on the backend too, but I think I'm going to wait until we redo storage since its kind of annoying having differing table names from model names -- will fix this change as its not intended